### ISSUE TITLE
[Bug Report] last release (v0.17.2) has broken CSS - dist/vuetify.css

### ISSUE THREAD

**Issue by MathieuDerelle** [vuetifyjs/vuetify#2623]
when using 
`import('path/to/node_modules/vuetify/dist/vuetify.min.css')`
we get the error 
Undefined variable: "$chip-label-color".

In fact, when exploring the node_modules, in the compiled CSS dist/vuetify.css, we can still see the sass variable : 
```
.chip {
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  border-radius: 28px;
  border: 1px solid transparent;
  color: $chip-label-color;
  display: -webkit-inline-box;
  display: -ms-inline-flexbox;
  display: inline-flex;
  font-size: 13px;
  margin: 4px;
  outline: none;
  position: relative;
  -webkit-transition: 0.3s cubic-bezier(0.25, 0.8, 0.5, 1);
  transition: 0.3s cubic-bezier(0.25, 0.8, 0.5, 1);
  vertical-align: middle;
}
```

**Comment by KaelWD** [comment]
https://github.com/vuetifyjs/vuetify/commit/38f84be0028220908bf4cc4cabcebfd6070773d8#diff-c0c919c2a7dfb89af1fd7c781f35ee51

**Comment by KaelWD** [comment]
This has already been fixed in 1ea6586a538f3ac2ff4cc2956bb345599382866d
